backend/entities/AccessRequest.cs

```diff
@@ -13,7 +13,7 @@ public class AccessRequest : BaseEntity
         /// get/set - The primary key IDENTITY.
         /// </summary>
         /// <value></value>
-        public Guid Id { get; set; }
+        public int Id { get; set; }
```

Comment:
Guid's are generally bad primary keys as they will cause defragmentation and need the index to be rebuilt constantly.